manifests/init.pp

```diff
@@ -363,7 +363,7 @@
         # Added for Amazon Linux support
         # Amazon Linux has 'YYYY-MM' version format
         # https://github.com/ghoneycutt/puppet-module-nscd/issues/41
-        '2015', '2016': {
+        /^201[5-7]+$/: {
```

Comment:
Could you please add 2017 to the README under Compatibility.

https://github.com/ghoneycutt/puppet-module-nscd#compatibility